device/chip/tlb_manager.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-FileCopyrightText: (c) 2024 Tenstorrent Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
+
+#include "umd/device/tt_xy_pair.h"
+#include "umd/device/types/tlb.h"
+
+namespace tt {
+class Writer;
+}
+
+namespace tt::umd {
+
+class TTDevice;
+
+class TLBManager {
+public:
+    TLBManager(TTDevice* tt_device);
+
+    void configure_tlb(tt_xy_pair core, int32_t tlb_index, uint64_t address, uint64_t ordering);
+
+    void set_dynamic_tlb_config(std::string fallback_tlb_name, int32_t tlb_index);
+    void set_dynamic_tlb_config_ordering(std::string fallback_tlb_name, uint64_t ordering);
```

Comment:
Valid fallback_tlb_names?

Reply:
What is the comment? The function does check whether this is a valid fallback tlb.
Or is the comment around naming?
This should be converted to an enum obviously.

Reply:
Let me know if there's additional comment here.